Userland/Libraries/LibWeb/Painting/PaintableBox.h

```diff
@@ -157,7 +157,7 @@ class PaintableBox : public Paintable {
     Optional<CSSPixelRect> mutable m_absolute_rect;
     Optional<CSSPixelRect> mutable m_absolute_paint_rect;
 
-    Optional<CSSPixelRect> mutable m_clip_rect;
+    Gfx::IntRect mutable m_clip_rect;
```

Comment:
maybe leaking clip rectangle from painter to paintable box is not the best thing to do to solve the problem that painting of out-of-flow boxes need to continue using clip rectangle of their containing block. probably changing painter to allow to restore state from saved earlier object might be better option than saving just clip rectangle slice.